free-1.md

```diff
@@ -9,42 +9,141 @@ good way to meet their needs.
+
+Thus, I propose a DSL that is a markup language specifically for economics to
+allow transcribing notes efficiently, but most importantly, allow for graphs to
+be embedded in the notes very quickly and easily.  This allows a economics
+student to have their notes all in one place, and the opportunity to transcribe
+the verbal lecture from the professor while having relevant graphs available for
+reference.  
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+A DSL is appropriate because lecture notes consists of content written in some
+language. Thus, the solution to the problem above should also be some sort of
+language. Furthermore, since notes can be taken on the computer, a computer
+programming language makes sense; however, it serves a very specific focus, so
+it should be a domain-specific language so it does not confuse economics students
+who likely have little interest in learning a massive language for this small
+purpose. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
-
+This excites me because it stems out of a personal problem. My high school was
+okay with allowing me to take notes on the computer in any class. I found
+that social sciences classes which require memorization, teachers
+who did not necessarily write every important definition or point out but rather
+preferred to talk about it, could be extremely frustrating if good notes weren't
+taken. I became a great note taker with my computer and, though studies show
+otherwise, felt like I absorbed more through taking robust, detailed notes on 
+the computer rather than listening and hand-writing notes sparingly. The biggest
+issue came in college economics, my major, when many graphs came into play and
+therefore made computer notes very difficult.  Using Evernote, clipping pictures
+of the relevant graphs from Google images was the only solution, but was too
+inefficient to continue doing.  So, I thought having a domain specific language
+for this would be a great solution. 
 
 ### Domain
 _Describe the project's domain in five words._
-
+Creating graphs for economics notes
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
-
+The user would likely be using some sort of text editor or word processor for
+taking notes. This is an early problem for me: can I create the language to be
+usable in popular word processors? Regardless, I would imagine some note snippets
+to look similar to the following in this language:
+
+``` 
+This is a standard supply and demand graph
+
+Graph{
+	Y-Axis: Price
+	X-Axis: Quantity
+	SupplyCurve()	
+	DemandCurve()
+}
+
+generate Graph
+```
+
+```
+Today, we're learning about the *Engel Curve*, the graph looks like this: 
+
+Graph{
+	Y-axis: Income/month
+	X-axis: Units
+	EngelCurve()
+}
```

Comment:
This language really looks like JSON - honestly, I think you could accomplish this task pretty easily by just embedding JSON in a document and having some library that understands the format of the JSON. Seems like using a full DSL might be overkill.